src/maintenance.js

```diff
@@ -194,9 +193,10 @@ async function updateQueueStatus(name, currentState, RabbitQueue) {
 
 async function handleQueues({cfg, prefix, manager, namespaces, RabbitQueue, notify, virtualhost}) {
   let debug = Debug('maintenance.handle-queues');
-  await Promise.map(await manager.queues(virtualhost), async queue => {
+  let queues = await manager.queues(virtualhost);
+  for (let queue of queues) {
```

Comment:
Bluebird's `Promise.map` had a `concurrency` option, so we would do these 10 at a time.  This is a maintenance chore, though, so we can do them sequentially too and keep overall load down.